engines/wage/entities.cpp

```diff
@@ -159,6 +159,13 @@ void Scene::paint(Graphics::ManagedSurface *surface, int x, int y) {
 Designed *Scene::lookUpEntity(int x, int y) {
 	for (ObjList::const_iterator it = _objs.end(); it != _objs.begin(); ) {
 		it--;
+		if (((*it)->_name == "continue" && (*it)->_resourceId == 22259) ||
+			((*it)->_name == "Goth" && (*it)->_resourceId == 18623))
+			if (_design->isPointOpaque(x, y))
+				return *it;
+
+		if ((*it)->_name == "TWIST.CLICK")
```

Comment:
Same thing, please explain in the comment what does it do.

Alll these comments need to start from "WORKAROUND: "

Reply:
Done